extension/src/plots/vega/util.test.ts

```diff
@@ -139,7 +139,7 @@ describe('extendVegaSpec', () => {
 
   it('should truncate all titles from the left to 50 characters for large plots', () => {
     const spec = withLongTemplatePlotTitle()
-    const updatedSpec = extendVegaSpec(spec, 500)
+    const updatedSpec = extendVegaSpec(spec, PlotSizeNumber.LARGE)
```

Comment:
This was the source of my weird failing tests. Shows the importance of always using variables, because when they change, that makes it so much harder to find where they are used.